ert3/console/__init__.py

```diff
@@ -123,12 +135,17 @@ def main():
         _init_workspace(Path.cwd())
         return
 
+    pm = ErtPluginManager()
```

Comment:
Do we need this one?

Reply:
Not really I suppose, my intent was to show how different ways of loading scripts can be done, but might be overkill at this point.